bucket/w64devkit.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.6.0",
+    "description": "Portable C and C++ Development Kit for x64 Windows",
+    "homepage": "https://github.com/skeeto/w64devkit",
+    "license": {
+        "identifier": " BSD-style",
+        "url": "https://sourceforge.net/p/mingw-w64/mingw-w64/ci/master/tree/COPYING.MinGW-w64-runtime/COPYING.MinGW-w64-runtime.txt"
```

Comment:
Why there is link to some random sourceforge mingw repository??

![image](https://user-images.githubusercontent.com/13260377/108895809-57d60780-7614-11eb-8910-930cfdd0aca9.png)

Reply:
Probably because the devkit itself does not provide any licensed stuff. It's a docker image that packages some gnu and mingw tools.

Reply:
![image](https://user-images.githubusercontent.com/17216524/108896464-92bb4980-75da-11eb-96dd-4bf0038e5b78.png)
At the bottom of the readme he mentions a series of licenses. How would this best be handled?

Reply:
Updated License